docs/docsource/installation/upgrade/xdm_migrate_from_kubedb_to_bitnami.rst

```diff
@@ -0,0 +1,282 @@
+   the `Zenko` in a `DeploymentInProgress=True` state until the `MongoDB`
+   service becomes available. Additionally, the KubeDB MongoDB deployment will
+   be deleted, and the  `DormantDatabase` resources will be created.
+
+Releasing and removing claims on the `PV` resources
+---------------------------------------------------
+
+#. Make sure the `DormantDatabase` resources are in a ``Paused`` state.
+
+#. Once ``Paused``, delete the `DormantDatabases` resources:
+
+   .. code::
+
+      kubectl delete -n zenko dormantdatabase -l kubedb.com/kind=MongoDB
+
+#. Delete the `PVC` created by `KubeDB`:
+
+   .. code::
+
+      kubectl delete -n zenko pvc -l kubedb.com/kind=MongoDB,kubedb.com/name=artesca-data-base-db
+
+#. Remove the `claimRef` from each `PV` to make them `Available`:
+
+   - Retrieve the list of `PVs`:
+
+     .. code::
+
+        kubectl get pv -l app.kubernetes.io/name=mongodb,app.kubernetes.io/part-of=zenko
+
+   - Remove `claimRef` from each `PV` in the list:
+
+     .. code::
+
+        kubectl patch pv <pv name> --type merge -p '{"spec":{"claimRef": null}}'
+
+   The `Pending` Bitnami MongoDB deployment should now be able claim the `PV`
+   resources and continue deploying the `MongoDB` resources.
+
+Configuring the MongoDB Replica Set
+-----------------------------------
+
+#. Wait for the `mongodb-primary` and `mongodb-secondary` statefulsets to be
+   successfully deployed.
+
+#. Access the `mongo` shell to start configuration procedure:
+   
+   .. code::
+
+      MONGODB_ROOT_USERNAME=$(kubectl -n zenko get secret mongodb-db-creds -o jsonpath='{.data.mongodb-root-username}' | base64 -d)
+      MONGODB_ROOT_PASSWORD=$(kubectl -n zenko get secret mongodb-db-creds -o jsonpath='{.data.mongodb-root-password}' | base64 -d)
+      kubectl -n zenko exec -ti mongodb-primary-0 -c mongodb-primary -- mongo -u $MONGODB_ROOT_USERNAME -p $MONGODB_ROOT_PASSWORD
+
+#. In `mongo` shell, run the following commands:
+
+   - Create `rsconf` variable in:
+
+     .. code::
+
+        rs0:OTHER> rsconf = rs.conf()
+
+   - Add replica set members: 
+ 
+     .. code::
+
+        rs0:OTHER> rsconf.members = [
+        ... {_id: 0, host: "mongodb-primary-0.mongodb-headless.zenko.svc.cluster.local:27017"}
+        ... ] 
+
+     If running 3-nodes, the following should be added:
+
+     .. code::
+
+        rs0:OTHER> rsconf.members = [
+        ... {_id: 0, host: "mongodb-primary-0.mongodb-headless.zenko.svc.cluster.local:27017"},
+        ... {_id: 1, host: "mongodb-secondary-0.mongodb-headless.zenko.svc.cluster.local:27017"},
+        ... {_id: 2, host: "mongodb-secondary-1.mongodb-headless.zenko.svc.cluster.local:27017"}
+        ... ] 
+
+   - Run `rs.reconfig` to elect new `PRIMARY`:
+
+     .. code::
+
+        rs0:OTHER> rs.reconfig(rsconf, { force: true})
+
+After running these steps, the MongoDB replica set will be configured, and the Zenko
+services should be able to connect to the database successfully.
```

Comment:
A command that waits until everything is ok would be useful (possibly with a timeout of a few minutes).

On error (typically timeout), a list of steps to identify what failed would be good, since it's going to be the first thing asked in support requests.

Reply:
* added commands to check status of zenko as the final step
* added commands for determining what the error is